src/app/state/store.ts

```diff
@@ -18,6 +19,7 @@ export const store = configureStore({
     accountHistory: accountHistorySlice.reducer,
     priceInfo: priceInfoSlice.reducer,
     i18n: i18nSlice.reducer,
+    rewardSlice: rewardSlice.reducer,
```

Comment:
To match our naming convention:
```suggestion
    reward: rewardSlice.reducer,
```

Important: when doing this change, you will need to change this every time it was imported as well.

Reply:
Lets make this change as part of the bigger state change.